### DISCUSSION TITLE
Unable to show searchbar in taskbar

### DISCUSSION THREAD

**Discussion by SnakeSol** [valinet/ExplorerPatcher#3277] [General]
I just installed both last versions of Windows 11 and EP, and I can't show the entire searchbar in the taskbar, I just can hide it completely or show the icon.

Windows 11 Family edition
Version	23H2
Installed ‎2/22/‎2024
Build 22631.3593
Experience	Windows Feature Experience Pack 1000.22700.1003.0

EP build: 22621.3527.65.2

I only install stable versions of EP

Screenshot of "taskbar" section of parameters, and taskbar itself:
[image](https://github.com/valinet/ExplorerPatcher/assets/73792210/4647345e-86e6-4620-b5a3-d7ec5597f9b7)

**Comment by pyrates999** [comment]
Please include what settings you changed in explorer patcher properties application.  These should be screen shots.  If you can, please change it to english first.

Please include any other software you have installed that modifies the UI of windows.

Does it happen if you uninstall EP?

Try installing the latest pre-release of EP.

**Reply by Amrsatrio** [reply]
@pyrates999 This is caused by Microsoft nuking explorer.exe symbols out of their public symbol servers and I have no plans to make a workaround for this yet.

**Reply by pyrates999** [reply]
There was another issue that was filed where the EP console also showed unable to download symbols for explorer.exe.

**Reply by SnakeSol** [reply]
I think I should close this discussion. There's a problem, no fix planned, so what? Staying here to say there is other problems linked but "no plans to make a workaround yet"? That is useful for sure

**Reply by pyrates999** [reply]
This is the issue: https://github.com/valinet/ExplorerPatcher/issues/3227

Please close this.
